packages/action-icon-react/src/Icons/IconSearch.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+export const IconSearch = () => (
+    <svg
+        style={{ color: "currentColor" }}
```

Comment:
Inlinestyle?

Reply:
Yeah, that's my bad suggestion! 😅This can and should be set another way, maybe even in the component using the icon.

Reply:
But why only on the search icon, what is special about that? I do like inline styles every now and then.. but we might be able to solve this more elegantly.

Reply:
That one is on us, it was supposed to be implemented to all. Going to look into alternative solutions today amongst other things commented on in this PR.

Reply:
`fill=currentColor` i.e, or a scss file.